dev/storage_base.h

```diff
@@ -25,6 +25,7 @@
 #include "function.h"
 #include "values_to_tuple.h"
 #include "arg_values.h"
+// #include "final_action.h"
```

Comment:
please don't commit commented out code. Just remove it. If we need to restore some code we can use git for it

Reply:
ok, done!